python/ray/data/_internal/execution/interfaces/task_context.py

```diff
@@ -42,3 +46,36 @@ class TaskContext:
 
     # Additional keyword arguments passed to the task.
     kwargs: Dict[str, Any] = field(default_factory=dict)
+
+    @classmethod
+    def get_current(cls, create_if_not_exists=True, **kwargs) -> "TaskContext":
```

Comment:
nit, I don't think create_if_not_exists and kwargs are needed for this PR. 
Let's remove it.